src/ui/flutter_app/lib/screens/game_page/game_page.dart

```diff
@@ -68,129 +65,91 @@ class _GamePageState extends State<GamePage>
   final Engine _engine = NativeEngine();
 
   double screenPaddingH = AppTheme.boardScreenPaddingH;
-  final double boardMargin = AppTheme.boardMargin;
+  final double _boardMargin = AppTheme.boardMargin;
 
   late String _tip;
-  bool isReady = false;
-  bool isGoingToHistory = false;
-  late Timer timer;
+  bool _isGoingToHistory = false;
   late AnimationController _animationController;
-  late Animation<double> animation;
-  bool disposed = false;
+  late Animation<double> _animation;
+  bool _disposed = false;
   late bool ltr;
   static const String _tag = "[game_page]";
 
-  Future<void> _setReadyState() async {
-    debugPrint("$_tag Check if need to set Ready state...");
-    if (!isReady && mounted) {
-      debugPrint("$_tag Set Ready State...");
-      setState(() {});
-      isReady = true;
-      timer.cancel();
-
-      if (Localizations.localeOf(context).languageCode == "zh" &&
-          !LocalDatabaseService.preferences.isPrivacyPolicyAccepted) {
-        onShowPrivacyDialog();
-      }
-    }
-  }
-
   void _initAnimation() {
     _animationController = AnimationController(
       vsync: this,
       duration: Duration(
-        milliseconds:
-            (LocalDatabaseService.display.animationDuration * 1000).toInt(),
+        seconds: LocalDatabaseService.display.animationDuration.toInt(),
       ),
     );
-    _animationController.addListener(() {});
 
-    animation = Tween(
-      begin: 1.27, // sqrt(1.618) = 1.272
-      end: 1.0,
-    ).animate(_animationController)
+    // sqrt(1.618) = 1.272
+    _animation = Tween(begin: 1.27, end: 1.0).animate(_animationController)
       ..addListener(() => setState(() {}));
 
     _animationController.addStatusListener((state) {
       if (state == AnimationStatus.completed ||
           state == AnimationStatus.dismissed) {
-        if (disposed) {
-          return;
-        }
-        _animationController.forward();
+        if (!_disposed) _animationController.forward();
       }
     });
-
-    if (!disposed) {
-      _animationController.forward();
-    }
   }
 
-  void showTip(String tip) {
+  void _showTip(String tip, {bool snackBar = false}) {
     if (!mounted) return;
 
     debugPrint("[tip] $tip");
-    if (LocalDatabaseService.preferences.screenReaderSupport) {
-      //showSnackBar(context, tip);
+    if (LocalDatabaseService.preferences.screenReaderSupport && snackBar) {
+      ScaffoldMessenger.of(context).clearSnackBars();
```

Comment:
The changes here seem to be contrary to the original logic. According to the original design, when blind people use this app, some sentences displayed by the snack bar must be converted into speech and cannot be interrupted midway. But some sentences can be interrupted. So I distinguished them, rather than interrupt them here. Blind friends have confirmed this design.